pennylane_lightning/core/src/simulators/lightning_qubit/tests/Test_StateVectorLQubit.cpp

```diff
@@ -276,9 +275,7 @@ TEMPLATE_PRODUCT_TEST_CASE("StateVectorLQubit::applyMatrix with a pointer",
             const auto m = randomUnitary<PrecisionT>(re, num_wires);
 
             state_vector_1.applyMatrix(m, wires);
-
-            Gates::GateImplementationsPI::applyMultiQubitOp<PrecisionT>(
-                state_vector_2.getData(), num_qubits, m.data(), wires, false);
+            state_vector_2.applyMatrix(m, wires);
```

Comment:
Could we just remove this test? The results will match always.

Reply:
Yeah, let's remove it!